README.adoc

```diff
@@ -145,306 +136,184 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage and in
-server directory. 
+[role="code_command hotspot hotspot file=0", subs="quotes"]
+----
+#Update the `PropertiesResource` class.#
+`src/main/java/io/openliberty/guides/rest/PropertiesResource.java`
+----
 
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v $(pwd)/target/liberty/wlp/usr/servers:/servers -u `id -u` ol-runtime
-```
---
+[role="edit_command_text"]
+Change the endpoint of your application from `properties` to `properties-new` by changing the [hotspot=Path]`@Path`
+annotation to `"properties-new"`.
 
-Alternatively, you can also execute the `run` command to mount a single server instead of the whole `servers` directory:
-[role='command']
+After you make the file changes, Open Liberty automatically updates the application. To see these changes reflected in the application, 
+point your browser to http://localhost:9080/system/properties-new[http://localhost:9080/system/properties-new^].
 
-include::{common-includes}/os-tabs.adoc[]
+// =================================================================================================
+// Testing the container 
+// =================================================================================================
 
-[.tab_content.windows_section]
---
-Make sure you substitute the absolute path to this guide.
+== Testing the container 
 
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v <absolute path to guide>/start/target/liberty/wlp/usr/servers/defaultServer:/servers/defaultServer ol-runtime
-```
---
+// File 0
+EndpointIT.java
+[source, java, linenums, role='code_column hide_tags=comment']
+----
+include::finish/src/test/java/it/io/openliberty/guides/rest/EndpointIT.java[]
+----
 
+You could test this service manually by starting a server and pointing a web browser to the 
+http://localhost:9080/system/properties-new[http://localhost:9080/system/properties-new^] URL. 
+However, automated tests are a much better approach because they trigger a failure if a change introduces a bug.
+JUnit and the JAX-RS Client API provide a simple environment to test the application. 
+You can write tests for the individual units of code outside of a running application server, or they
+can be written to call the application server directly. In this example, you will create a test that does the latter.
 
-[.tab_content.mac_section]
---
-[role='command']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v $(pwd)/target/liberty/wlp/usr/servers/defaultServer:/servers/defaultServer ol-runtime
-```
---
+[role="code_command hotspot file=0", subs="quotes"]
+----
+#Create the `EndpointIT` class.#
+`src/test/java/it/io/openliberty/guides/rest/EndpointIT.java`
+----
 
+This test make a request to the `/system/properties-new` endpoint and checks to
+make sure that the response has a valid status code, and that the information in
+the response is correct. 
```

Comment:
Tests fail for me, as app seems to run on port 9081 as previously mentioned.

Reply:
Resolved, had an application on my machine running on port 9080 for some reason. Terminated this application and restarted guide app. Everything worked as expected.